HousingRepairsOnlineApi/Helpers/NotificationConfiguration/BaseNotificationConfigurationProvider.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Ardalis.GuardClauses;
+using HousingRepairsOnlineApi.Domain;
+using HousingRepairsOnlineApi.UseCases;
+
+namespace HousingRepairsOnlineApi.Helpers.SendNotifications;
+
+public class BaseNotificationConfigurationProvider
+{
+    public string ConfirmationSmsTemplateId { get; set; }
```

Comment:
Each of these properties has a public setter, which doesn't seem to be used.
If they are not required externally of this class make them private (or alternatively use [Init only setters](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/init)).

Reply:
init only setters - what a nice feature. I've implemented it using init only setters. These are needed to be accessed from outside the class.

Reply:
So I kept them public but through the getter.